addons/payment/wizards/onboarding_payment_wizard.py

```diff
@@ -0,0 +1,118 @@
+from odoo import api, models, fields
+
+
+class PaymentWizard(models.TransientModel):
+    _name = 'onboarding.payment.wizard'
+
+    payment_methods = 'paypal stripe'.split()
+    payment_modules = ['payment_' + n for n in payment_methods]
+
+    config_fields = {
+        'paypal': 'paypal_email_account paypal_seller_account paypal_pdt_token'.split(),
+        'stripe': 'stripe_secret_key stripe_publishable_key'.split(),
+    }
+    manual_fields = 'name post_msg'.split()
+
+    data_fetched = False
+    cache = {}
+
+    def _get_company_id(self):
+        return self.env.context['company_id']
+
+    def _get_company(self):
+        return self.env['res.company'].browse(self._get_company_id())
+
+    def _get_manual_payment(self, env=None):
+        if env is None:
```

Comment:
Why trying to pass an environment ? You already have one on self .

Reply:
This method is called 2 times, one time with a new env different from the current one.

Reply:
I saw that, but why do you need to instantiate a new environment ? It shouldn't be the case. If you need to create a new environment because you just installed a module, then refresh the page instead.